.github/workflows/GnuTests.yml

```diff
@@ -77,14 +77,14 @@ jobs:
       run: |
         ## Build binaries
         cd '${{ steps.vars.outputs.path_UUTILS }}'
-        bash util/build-gnu.sh
+        bash util/build-gnu.bash
```

Comment:
why this change?
why not making them .sh compatible instead?

Reply:
Yeah, I just jumped to the rename for `shellcheck`.

But I think it's just a variable expansion, so it should be reasonably simple to change. 
I'll look at it from that angle and try to rework it as POSIX-shell compatible.

Reply:
Yep, dropping this rename (as unneeded) in the revision.
I must have been using an old `shellcheck` on a different PC; checking now shows no complaints.